sirepo/template/raydata.py

```diff
@@ -121,10 +123,14 @@ def stateless_compute_scans(data):
             raise sirepo.util.UserAlert(
                 f'More than {_MAX_NUM_SCANS} scans found. Please reduce your query.',
             )
-        s.append(_scan_info(v[0], metadata=v[1].metadata))
+        s.append(_scan_info(v[0], data.selectedColumns, metadata=v[1].metadata))
     return _scan_info_result(s)
 
 
+def stateless_compute_get_columns(data):
+    return PKDict(columns=list(databroker.catalog['csx'][-1].metadata['start'].keys()))
```

Comment:
Use `catalog()` instead of `databroker.catalog['csx']`

Reply:
e64969d8a3a9303ebcf97566971db0411515341f